### PR TITLE
[DRAFT] Remove Apple hooks

### PR DESCRIPTION
**Pull request by fortuna**
This simplifies the build.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -145,4 +145,4 @@
     "trailingComma": "es5",
     "printWidth": 120
   }
-}
+}
```
Comment (daniellacosse) [inline package.json:148]: Do we care about this? https://stackoverflow.com/a/729795

**Review by daniellacosse — APPROVED** [review]
Assuming the newline isn't an issue, LGTM

File: package.json (hunk 1 above)
Comment (fortuna) [reply package.json:148]: I think the style guide says we should have line breaks at the end, but it seems our auto formatting is fighting that...

**Comment by fortuna** [comment]
Ugh, I'm having issues with this PR.
It works if I manually add the plugin with --link. But not on a clean install and platform add from the package.json.
That's because cordova is copying the entire third_party tree following symlinks. Besides the unnecessary copies, it breaks the ios frameworks, as they rely on symlinks.
Perhaps I can change the build to call `platform add --link` for the outline plugin instead of loading from the config, but it starts to be hacky again...

**Review by jyyi1 — APPROVED** [review]

**Comment by fortuna** [comment]
Deprecated by https://github.com/Jigsaw-Code/outline-client/pull/1527

**Pull request closed without merging**